src/simulator/halo_simulator.py

```diff
@@ -228,12 +229,40 @@ def simulated_venn_diagram_reach_by_spend(
               satisfying the request.
             max_frequency:  The maximum frequency for which to report reach.
         Returns:
-            A list of ReachPoint. Each reach point represents the mapping from 
+            A list of ReachPoint. Each reach point represents the mapping from
             the spends of a subset of publishers to the differentially private
             estimate of the number of people reached in this subset.
         """
         raise NotImplementedError()
 
+    def _aggregate_reach_in_venn_diagram_regions(
+        self, pub_ids: List[int], venn_diagram_regions: Dict[int, List]
+    ) -> int:
+        """Returns an aggregated reach from Venn diagram regions given publishers.
```

Comment:
given a subset of publishers

Reply:
I was trying to have a short description that can fit in one line as @matthewclegg suggested [here](https://github.com/world-federation-of-advertisers/planning-evaluation-framework/pull/49#discussion_r670122542). Do you have any suggestions?

Reply:
I see. It makes sense to drop some words to fit it in a line.